includes/discovery/sensors/temperatures/ceraos.inc.php

```diff
@@ -1,16 +1,16 @@
 <?php
 
-if ($device['os'] == 'ceragon') {
+if ($device['os'] == 'ceraos') {
     $mib  = 'MWRM-UNIT-MIB::genEquipUnitIduTemperature.0';
     $oid = ' .1.3.6.1.4.1.2281.10.1.1.9.0';
     $oids = snmp_get($device, $mib, '-OsqnU', 'MWRM-UNIT-MIB');
```

Comment:
In terms of code, you prefer the $mib without the MWRM-UNIT-MIB:: or to remove it from last parameter of snmp_get?